### PR TITLE
TASK: Markdown processing

### PR DESCRIPTION
**Pull request by mficzel**
This will convert the markdown from the chatbot response to html ...

### CODE REVIEW

File: Classes/Domain/ContentCollection.php [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    {
+        $converter = new CommonMarkConverter([
+            'html_input' => 'strip',
+            'allow_unsafe_links' => false,
+        ]);
+
+        $contents = [];
+        foreach ($response->content as $id => $textOrImage) {
+            if ($textOrImage instanceof ThreadMessageResponseContentTextObject) {
+                $html = $converter->convert($textOrImage->text->value);
```
Comment (nezaniel) [inline Classes/Domain/ContentCollection.php]: I'd move this into a ContentText::fromThreadMessageResponseContentTextObject (or similar) method to be able to instantiate the elements without the necessity of being part of a collection

File: Classes/Domain/ContentText.php [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+#[Flow\Proxy(false)]
+final class ContentText
+{
+    public function __construct(
+        public readonly string $value,
+    ) {
+    }
+
+    public function getType(): string
```
Comment (nezaniel) [inline Classes/Domain/ContentText.php:30]: What is this method used for?

**Review by nezaniel — CHANGES_REQUESTED** [review]

File: Classes/Domain/ContentText.php (hunk 2 above)
Comment (mficzel) [reply Classes/Domain/ContentText.php:30]: Damit man im fusion `@if={content.type == 'text'}` machen kann

File: Classes/Domain/ContentCollection.php (hunk 1 above)
Comment (mficzel) [reply Classes/Domain/ContentCollection.php]: Adjusted

**Review by nezaniel — APPROVED** [review]

**Pull request merged**